app/uploaders/upload_uploader.rb

```diff
@@ -0,0 +1,71 @@
+class UploadUploader < CarrierWave::Uploader::Base
```

Comment:
This file seems largely duplicative of `app/uploaders/footer_image_uploader.rb` and `app/uploaders/splash_image_uploader.rb`. If it isn't an errant copy, is there low-hanging fruit for abstraction?

Reply:
59fbd54